core/controllers/resources.py

```diff
@@ -86,3 +87,18 @@ def get(self, exploration_id, asset_type, encoded_filename):
             self.response.write(raw)
         except:
             raise self.PageNotFoundException
+
+
+class PromoBarHandler(base.BaseHandler):
+    """The handler for checking if Promobar is enabled and fetching
+    promobar message.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        self.render_json({
+            'promo_bar_enabled': config_domain.PROMO_BAR_ENABLED.value,
+            'promo_bar_message': config_domain.PROMO_BAR_MESSAGE.value
```

Comment:
If `config_domain.PROMO_BAR_ENABLED` doesn't exist, on starting the server following error occurs: `AttributeError: 'module' object has no attribute 'PROMO_BAR_ENABLED'`.

Reply:
So you should handle the cases when `config_domain.PROMO_BAR_ENABLED` doesn't exist so that the handler returns:
```
'promo_bar_enabled': False,
'promo_bar_message': ""
```

Reply:
Done.